### ISSUE TITLE
Voice aliases

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4433]
**Reported by td.dinakar on 2014-09-01 06:54**
While reading text line by line or by using say all command, NVDA reports the elements information such as heading, link,, etc. It is quite annoying to hear the words "link" and "Heading level 1", etc., each and every time. If we disable reporting of these elements in the settings, we may not be able to know whether a particular text is a link or heading.

 In JAWS, we have a good feature called "Speech Aliases", which help us to set different voices for different elements. When this feature is enabled, the screen reader reads only the text of the heading or the link, but with a different voice, so that we can identify the element of the text.

Since the default eSpeak synthesizer used in NVDA has several voice variants, it will be very useful if voice aliases are also made available in NVDA for reading the web elements. 

**Blocked by #4877**
**Blocking #5461**

**Comment by zahra21** [comment]
hi.
i have question.
i dont want to use different voice for different elements.
but i only wish that only use say all with no informations about elements and use arrow keys and quick navigation keys with elements.
for example: when i want to use say all command, i only wish that read my document or book without such informations,
but when i use navigation quick keys or arrow keys, i wish that hear these informations!
i asked in mailing list for users and suggested me configuration profile and trigggers.
but it is not my desire and wantedd suggestion, because in this case i cant realize for example even links using navigation quick and arrow keys!
what should i do to achieve my Goal?
thanks extremely for your help and God bless you all.

**Comment by Brian1Gaff** [comment]
Well you could make it manually triggered so you can turn it on and off when 
you want to.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by cary-rowen** [comment]
I think this is a useful feature. After such a long time in the past, I still hope that this issue gets attention and I hope to increase its priority.

**Comment by Adriani90** [comment]
very related to #4086.

**Comment by Adriani90** [comment]
This is a use case for the framework inmplemented in #7599.

**Comment by Adriani90** [comment]
Closing as duplicate of #4086.
